packages/expo-modules-core/android/src/main/java/expo/modules/kotlin/sharedobjects/SharedObject.kt

```diff
@@ -10,6 +14,32 @@ open class SharedObject {
    */
   internal var sharedObjectId: SharedObjectId = SharedObjectId(0)
 
+  internal var appContextHolder = WeakReference<AppContext>(null)
```

Comment:
Just wanted to note that access to the app context on iOS is public (setter is internal)

Reply:
I can add a public getter that automatically unwraps the app context. I don't want to give access to the weak reference here.